packages/owl-bot/src/owl-bot.ts

```diff
@@ -64,6 +64,19 @@ async function acquireLock(target: string): Promise<DatastoreLock> {
   return lock;
 }
 
+async function releaseLock(lock: DatastoreLock): Promise<void> {
+  try {
+    await lock.release();
+  } catch (_err: unknown) {
+    const err = _err as Error;
```

Comment:
When I've written these try/catch blocks in typescript, my IDE infers that the caught exception is an Error -- is this more save?

Reply:
At some point typescript changed the type of caught errors to `unknown` or `any`, which is kind of annoying.